_sass/_components/_footer.scss

```diff
@@ -1,9 +1,16 @@
+$footer-max-width: 1440px;
+
 .footer {
   background: $grey;
   padding: 3rem 0 2rem 0;
   color: $white;
   font-weight: 600;
 
+  > .container-fluid {
+    max-width: $footer-max-width;
+    margin-left: 0;
```

Comment:
shouldn't be centered?
`margin-left: auto` + `margin-right: auto` ?

Reply:
We don't center the footer, we keep the footer links to the left in other apps. Check https://tools.debtcollective.org/.

Since we are migrating to React components, we should abstract this header and the footer so we can reuse them in other apps. Maybe as a future improvement.